skipruntime-ts/wasm/package.json

```diff
@@ -9,7 +9,7 @@
     }
   },
   "scripts": {
-    "build": "node ../../prebuild.mjs skipwasm-std skipwasm-json && tsc && skargo build -r --target wasm32-unknown-unknown --lib --manifest-path=../native/Skargo.toml --out-dir=./dist/src/",
+    "build": "skargo clean -r --target wasm32-unknown-unknown --manifest-path=../native/Skargo.toml && node ../../prebuild.mjs skipwasm-std skipwasm-json && tsc && skargo build -r --target wasm32-unknown-unknown --lib --manifest-path=../native/Skargo.toml --out-dir=./dist/src/",
```

Comment:
Was this `clean` a workaround for an issue that remains, or does it make sense that it is needed?